src/internal/analytics/interfaces.ts

```diff
@@ -45,20 +40,14 @@ export type FunnelStartMethod = (props: FunnelStartProps) => string;
 
 // Define individual method props by extending the base
 export interface FunnelStepProps extends BaseFunnelProps {
-  instanceIdentifier?: AnalyticsMetadata['instanceIdentifier'];
+  stepIdentifier?: AnalyticsMetadata['instanceIdentifier'];
   stepNumber: number;
   stepName?: string | undefined;
-  stepNameSelector: string;
+  stepNameSelector?: string;
```

Comment:
This changes to optional as you can provide the stepName directly in the Wizard and so do not need the selector.